Dockerfile

```diff
@@ -1,101 +1,46 @@
 # Creating a python base with shared environment variables
-FROM python:3.10.5-slim as python-base
-ENV PYTHONPATH=/app \
-    PYTHONUNBUFFERED=1 \
-    PYTHONDONTWRITEBYTECODE=1 \
-    PIP_NO_CACHE_DIR=off \
-    PIP_DISABLE_PIP_VERSION_CHECK=on \
+FROM python:3.10.5 as base
+ENV PIP_NO_CACHE_DIR=off \
     PIP_DEFAULT_TIMEOUT=100 \
+    PIP_DISABLE_PIP_VERSION_CHECK=on \
     POETRY_HOME="/opt/poetry" \
-    POETRY_VIRTUALENVS_IN_PROJECT=true \
     POETRY_NO_INTERACTION=1 \
-    PYSETUP_PATH="/opt/pysetup" \
-    VENV_PATH="/opt/pysetup/.venv" \
-    PORT=8000
```

Comment:
This is necessary I think.

I ran `make shell`  from a fresh environment, and got this:

```
[2/2] STEP 11/13: EXPOSE $PORT
ERROR: Service 'web' failed to build : error building at STEP "EXPOSE $PORT": EXPOSE requires at least one argument

make: *** [shell] Error 1
```